src/include/kernel/ext_manifest.h

```diff
@@ -73,6 +73,18 @@ enum ext_man_elem_type {
 	EXT_MAN_ELEM_FW_VERSION		= 0,
 	EXT_MAN_ELEM_CC_VERSION		= SOF_IPC_EXT_CC_INFO,
 	EXT_MAN_ELEM_PROBE_INFO		= SOF_IPC_EXT_PROBE_INFO,
+	EXT_MAN_ELEM_CONFIG_DATA	= 5,
+};
+
+/* EXT_MAN_ELEM_CONFIG_DATA elements identificators */
+enum config_elem_type {
+	EXT_MAN_CONFIG_CAVS_LPRO	= 0,
```

Comment:
@abonislawski while talking to @fredoh9 about his PR, we realized that since the elems array has a fixed size of EXT_MAN_CONFIG_LAST_ELEM, the array will have to populated with the right tokens to not parse invalid elems as CAVS_LPRO tokens. So I think the elem type should start with a 1 instead. What do you think?

Reply:
@ranj063 yes, looks like a real issue. Thanks for spotting this, I will change to 1

Reply:
I saw your change, I will update mine and test it again

Reply:
@fredoh9 thanks